content/docs/deployment/csminstallationwizard/src/index.html

```diff
@@ -183,6 +189,18 @@
                           <label for="volumeless-pods" class="form-check-label ms-2 text-dark" data-bs-toggle="tooltip" data-bs-placement="right" title="Enable/Disable, if set to true will enable CSM for Resiliency to ignore pods without persistent volume attached to the pod.">Ignore Volumeless Pods</label>
                         </div>
                       </div>
+                      <div class="was-validated row mt-3 child-level-one" id="array-connectionloss-threshold">
+                        <label for="array-threshold" class="col-sm-3 col-form-label" data-bs-toggle="tooltip" data-bs-placement="right" title="Gives the number of failed connection polls that will be deemed to indicate array connectivity loss. Should not be set to less than 3.">Array Connectivity ConnectionLoss Threshold</label>
```

Comment:
Gives the number of failed connection polls that will indicate array connectivity loss.

(remove "deemed to")